src/docs/development/packages-and-plugins/plugin-api-migration.md

```diff
@@ -74,27 +74,52 @@ The following instructions outline the steps for supporting the new API:
       // now automatically registers plugins.
     }
     ```
+    
+1. (Optional) If you removed `MainActivity.java`, update the `<plugin_name>/example/android/app/src/main/AndroidManifest.xml` to use `io.flutter.embedding.android.FlutterActivity`. For example:
```

Comment:
The step above this says to keep the `MainActivity`. I don't think it makes sense to suggest this unless we also suggest removing the MainActivity. We suggest removing it in other places and last we talked about it that is @matthew-carroll's recommendation, so I think it makes sense to keep this note but also change the above to say to delete the `MainActivity` entirely instead.

Reply:
I think it might be beneficial to keep the above step completely, meaning leave the `MainActivity`; for cases like add-to-app, or people who edited their MainActivity for other purpose. If we say remove the `MainActivity` completely, people might panic if they have their custom stuff in the `MainActivity`.
Also, this doc is only for migration, so I don't know if we should consider add-to-app users getting confused by it.